ep_transition/__init__.py

```diff
@@ -1,2 +1,2 @@
-NAME = 'energyplus_transition_tools'
-VERSION = '2.0.3'
+NAME = "energyplus_transition_tools"
```

Comment:
I'm a little surprised to see the quotes changing everywhere.  I don't have a problem with it, it just stems from my built-in behavior to try to use single quotes for short constant literals and double quotes for long strings and where I'll be doing interpolation.  I realize this isn't a necessary idiom in Python, and having double quotes makes it easier to put in apostrophes in the text and such, so it's fine.

Reply:
black is an **opiniated** code formatter. see https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings where they explain why they chose double quotes. Also note:

> If you are adopting Black in a large project with pre-existing string conventions (like the popular [“single quotes for data, double quotes for human-readable strings”](https://stackoverflow.com/a/56190)), you can pass `--skip-string-normalization` on the command line. This is meant as an adoption helper, avoid using this for new projects.

Reply:
black is good at reformating your code, and I think we want that at least: leave the formating to the tool (like clang-format would do), and stop worrying about it.